recipes/nextcloud-carnet/package.json

```diff
@@ -1,13 +1,15 @@
 {
   "id": "nextcloud-carnet",
   "name": "Carnet (Nextcloud)",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
The version bump should be `1.2.1` and not `1.3.0`

Reply:
Thanks for the clarification, done